source/_components/image_processing.facebox_face_detect.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Facebox Face Detect"
+description: "Count the number of faces in an image with Facebox."
+date: 2018-05-03 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```

Comment:
May be we should use the [Machinebox logo](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRm85lGp21SSy75XvcMxiJZlYj6O467ujWW3k505RMdkUSyfSZf).

Reply:
I’ve approached to request permission. It’s a bad idea to use logos without express permission

Reply:
I think we had this discussion earlier. All logos as properties of the respective legal owners...and HA does not claim otherwise.